syntax = "proto3";
package del_contact_request;

message DelContactRequest {
    string uid = 1;
}